.github/workflows/docker-update-images.yaml

```diff
@@ -0,0 +1,26 @@
+# Continuously build Rosetta
+name: Docker Update Images
+
+on:
+  schedule:
+    - cron: "0 9 * * 1" # once a week
+
+jobs:
+  update:
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Check and pin updates (if any)
```

Comment:
You might need to add setup python step? Although without dependencies it might just work tm